### ISSUE TITLE
Could not queue reset for A

### ISSUE THREAD

**Issue by z0mbieslayer** [falkirks/MineReset#120]
So i have looked at all of the other issues for this problem and they all suggest that the world isnt loaded. However I am having the same problem as them except my level IS loaded. In fact, it my main world. Any help would be greatly appreciated!

**Comment by falkirks** [comment]
This can be caused by three things 

- The world isn't loaded.
- The mine is already resetting.
- There is a problem with how the mine was setup (ie: the ratio data is bad).

Could you tell me the full command you used to set the mine? It would start with `mine set`.

**Comment by z0mbieslayer** [comment]
> This can be caused by three things
> 
> * The world isn't loaded.
> * The mine is already resetting.
> * There is a problem with how the mine was setup (ie: the ratio data is bad).
> 
> I am leaning toward it being the last one. Could you tell me the full command you used to set the mine? It would start with `mine set`.

Thanks for responding! I used the command `mine set 1 75 16 25`

**Comment by falkirks** [comment]
That should be `mine set A 1 75 16 25` if the name of your mine is "A".

**Comment by z0mbieslayer** [comment]
I might have, let me try again

**Comment by z0mbieslayer** [comment]
> That should be `mine set A 1 75 16 25` if the name of your mine is "A". Have you tried that?

Did that command, still doesn't work. Ive used this plugin before and it worked fine in the past, don't know why it's acting up now

**Comment by falkirks** [comment]
Hmm weird. Could the world have changed name since the mine was setup?

**Comment by z0mbieslayer** [comment]
I don’t think I did. I immediately tried to reset the mine after I set it. I can send you the server files if you’d like.

**Comment by Rubyliss** [comment]
i got the exact same problem.

**Comment by Rubyliss** [comment]
After doing some test i found out that the error only come on an downloaded world, for exemple i downloaded an pre build prison mine world made on the PC version of MC, on it i got the error. 
On an generated world by MultiWorld plugin the error is not there and the plugin work great

**Comment by Rubyliss** [comment]
Aaaaaand i just got it fixed using the MultiWorl command "Fix name" :)

( /mw fix [Name of your world] )

**Comment by Chuckeymomo** [comment]
I found the same issue but Multiworlds “fix name” does not work for the PocketMine version of Minecraft. Is there any way I could fix it without that command?

**Comment by Chuckeymomo** [comment]
Never mind I fixed it by copy and pasting the mines to a multiworld-generated world

**Comment by Chuckeymomo** [comment]
And that fixed the issue.
Thanks so much for this great plugin falkirks
😃

**Comment by ExileWeb** [comment]
See PR #129. After looking at the source and modifying my config the fix uses the Level getFolderName which means PC-imported worlds that are loaded can be reset. 

Don’t think it should affect PM worlds as they have the same name.
